package.json

```diff
@@ -22,9 +22,12 @@
     "private::build:js-test-standalone": "run-s \"private::build -- --mode development --config webpack.test.standalone.config.js\"",
     "private::build:js-test-watch": "run-s \"private::build -- --mode development --config webpack.test.config.js --watch\"",
     "private::build:py": "dash-generate-components src/dash-table/dash/DataTable.js dash_table -p package-info.json && cp dash_table_base/** dash_table/ && dash-generate-components src/dash-table/dash/DataTable.js dash_table -p package-info.json --r-prefix 'dash'",
+    "private::format.ts": "npm run private::lint.ts -- --fix",
+    "private::format.black": "black --exclude dash_table .",
     "private::host_js": "http-server ./dash_table -c-1 --silent",
-    "private::lint:ts": "tslint --project tsconfig.json --config tslint.json",
-    "private::lint:py": "flake8 --exclude=DataTable.py,__init__.py,_imports_.py dash_table tests && black --check --exclude dash_table .",
+    "private::lint.ts": "tslint --project tsconfig.json --config tslint.json",
+    "private::lint.flake": "flake8 --exclude=dash_table*,node_modules,venv",
```

Comment:
so excluding `dash_table_base` too? we can't lint that?

Reply:
https://github.com/plotly/dash-table/pull/720/commits/cdd830886f5229362089f1fbedc7fd6bb4a35e51